apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/Variable.kt

```diff
@@ -0,0 +1,3 @@
+package com.apollostack.compiler.ir
+
+data class Variable(val name:String, val type:String)
```

Comment:
nit: maybe keep each field in its own line to stay consistent with the other data classes?